pkg/services/featuremgmt/registry.go

```diff
@@ -658,14 +658,6 @@ var (
 			Owner:        grafanaDatavizSquad,
 			Expression:   "true", // enabled by default
 		},
-		{
-			Name:         "transformationsVariableSupport",
-			Description:  "Allows using variables in transformations",
-			FrontendOnly: true,
-			Stage:        FeatureStageGeneralAvailability,
-			Owner:        grafanaDatavizSquad,
-			Expression:   "true", // Enabled by default
-		},
```

Comment:
this has been enabled by default in production since https://github.com/grafana/grafana/pull/89111.

it's safe to remove now.